source/System.cpp

```diff
@@ -964,3 +966,10 @@ void System::Price::Update()
 {
 	price = base + static_cast<int>(-100. * erf(supply / LIMIT));
 }
+
+
+
+double System::InvisibleFenceRadius() const
```

Comment:
Functions should be defined in the same order as they are defined in the header file, which means this function shouldn't be below the price functions.